lib/providers/linkedin.js

```diff
@@ -0,0 +1,30 @@
+// Load modules
+
+var Crypto = require('crypto');
+
+// Declare internals
+
+var internals = {};
+
+
+exports = module.exports = function (options) {
+
+    return {
+        protocol: 'oauth2',
+        auth: 'https://www.linkedin.com/uas/oauth2/authorization',
+        token: 'https://www.linkedin.com/uas/oauth2/accessToken',
+        scope: ['r_fullprofile', 'r_emailaddress', 'r_network', 'r_contactinfo', 'rw_groups'],
+        scopeSeparator: ',',
+        profile: function (credentials, params, get, callback) {
+            var query = {
```

Comment:
Please add a newline after functions... this will follow our style guide http://hapijs.com/styleguide